examples/word_of_day.js

```diff
@@ -0,0 +1,80 @@
+'use strict';
+
+const libingester = require('libingester');
+const mustache = require('mustache');
+const rp = require('request-promise');
+const template = require('./word_of_day_template');
+const url = require('url');
+
+const home_page = 'http://www.thefreedictionary.com/_/archive.htm'; // Home section
+
+
+function ingest_article_profile(hatch, uri) {
+    return libingester.util.fetch_html(uri).then(($profile) => {
+        const base_uri = libingester.util.get_doc_base_uri($profile, uri);
+        const asset = new libingester.NewsArticle();
+
+        asset.set_canonical_uri(uri);
+
+        // use url library to pull out query string date object
+        const parts = url.parse(uri, true);
+        const modified_date = new Date(Date.parse(parts.query.d));
+
+        asset.set_last_modified_date(modified_date);
+        asset.set_section("word_of_day");
+
+        //Set title section
+        const date = parts.query.d;
+        const title = $profile('h1').first().text() + ": " + date;
```

Comment:
Try to keep items grouped a bit better (vars on top and logic in the middle). title/date here should probably be near the top of this method since they get set once and used at the end.